flutter-studio/src/io/flutter/profiler/GcStatsDataSeries.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.profiler;
+
+import com.android.tools.adtui.model.DataSeries;
+import com.android.tools.adtui.model.Range;
+import com.android.tools.adtui.model.SeriesData;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.jetbrains.annotations.NotNull;
+
+public final class GcStatsDataSeries implements DataSeries<GcDurationData> {
+  FlutterAllMemoryData.ThreadSafeData gcData;
```

Comment:
Can you make this final?

Reply:
No one else is subclassing these type of classes - right?